### ISSUE TITLE
A11y: Add an H1 to the About/Overview page

### ISSUE THREAD

**Issue by bacitracin** [CodeForPhilly/vacant-lots-proj#327]
## Describe the task
It's not a requirement, but as a best practice, all pages should have an H1 to help screenreader users navigate the page.

There's an H1 on most pages of the app currently. The only ones missing it are the About/Overview and the Find properties page. 

In this ticket, add an H1 to the overview page at https://www.cleanandgreenphilly.org/about. There is a [separate ticket for the Find properties page](https://github.com/CodeForPhilly/vacant-lots-proj/issues/302).

<img width="666" alt="Screenshot 2024-02-10 at 12 07 03 PM" src="https://github.com/CodeForPhilly/vacant-lots-proj/assets/7329799/85cd9243-baf2-4756-b8f6-15b345bb2e92">

## Acceptance Criteria

- [ ] The page https://www.cleanandgreenphilly.org/about will have an appropriate H1 added to the page.

## Additional context

* Some more info about headings https://usability.yale.edu/web-accessibility/articles/headings

**Comment by bacitracin** [comment]
New About design includes an H1, this ticket is no longer relevant. Closing.
